Gemfile

```diff
@@ -18,7 +19,8 @@ gem 'nilify_blanks'
 gem 'nokogiri'
 gem 'pg'
 gem 'pg_search'
-gem 'rails', '3.2.22.5'
+gem 'protected_attributes'
```

Comment:
We could upgrade every model and controller to use strong params instead, if you'd rather?